Cargo.toml

```
@@ -20,6 +21,7 @@ criterion = "0.5"
 [features]
 default = []
 image = [ "dep:image" ]
+pixbuf = [ "dep:gdk-pixbuf" ]
```

```suggestion
gdk-pixbuf = [ "dep:gdk-pixbuf" ]
```

I'd say, keep the flag consistent.